main.js

```diff
@@ -7,6 +7,30 @@ var edges = {}; // global graph for inspection with aim to solve #1 issue on git
 const DEBUG = false;
 const STORAGE_KEY = 'kafka-streams-viz';
 
+var x = document.getElementById("ServersSelector");
+
+const servers = [
+	server("localhost:9973", "fulltext"),
+	server("localhost:9900", "joiner"),
+	server("localhost:9978", "debezium")
+]
+
+function createOption(server) {
+	var option = document.createElement("option");
+	option.text = server.name;
+	option.value = server.location;
+	x.add(option);
+}
+
+function selectedServers() {
+	var ddd = [];
+	for (var i = 0; i < x.selectedOptions.length; i++) {
+		 var s = x.selectedOptions[i];
+		ddd.push(server(s.value, s.text));
```

Comment:
This function relies on data stored within the `<option>` element. It should rather take all the data from the `servers` global variable.